lib/optimizely/config_manager/static_project_config_manager.rb

```diff
@@ -39,5 +38,9 @@ def initialize(datafile, logger, error_handler, skip_json_validation)
         skip_json_validation
       )
     end
+
+    def get_config
```

Comment:
I think, this should be defined in the projectconfigmanager as an abstract method. right now i can see config is defined in the projectconfig manager.